cpp/ps0007/ps0007_sol.cbp

```diff
@@ -54,3 +58,4 @@ int main() {
 
```

Comment:
- [x] checked the user's input for choosing the shape
- [x] filled in the dimensions array with positive numbers 
- [x] invoked function to calculate the area of the shape
- [x] used suitable size of dimensions arrays with the shape
- [x] used `cout::std` in a right way to print out the area